plugins/searchonline.koplugin/handleurlstring.lua

```diff
@@ -0,0 +1,171 @@
+local InfoMessage = require("ui/widget/infomessage")
+local UIManager = require("ui/uimanager")
+local url = require("socket.url")
+local util = require("util")
+local _ = require("gettext")
+
+local HandleUrlString = {}
+
+-- Message box to inform the user about a misformatted URL
+function HandleUrlString:urlError(text_explaining_error)
+    UIManager:show(InfoMessage:new{
+        text = _(text_explaining_error),

```

Comment:
I think this won't work, we need a string literal `_("something")` for it to be proposed to translators.
@Frenzie

Reply:
Correct.